pybikes/data/bysykkelgraphql.json

```diff
@@ -0,0 +1,19 @@
+{
+    "instances":[
+        {
+            "tag":"trondheim-bysykkel",
+            "meta":{
+                "latitude":63.4307238,
+                "country":"NO",
+                "name":"Trondheim Bysykkel",
+                "longitude":10.3936781,
+                "city":"Trondheim",
+                "city_code":"TRD",
+                "systemId": "trondheim"
+            },
+            "feed_url":"https://core.urbansharing.com/public/api/v1/graphql"
```

Comment:
Seems to me this URL is going to be common to any system on this feed, and the only thing changing would be the `systemId` right?

I would move `systemId` to the JSON level where `feed_url` currently is, and move `feed_url` directly into the python code.

Reply:
Yes, I think so. Will change it! :smile: